src/shared/components/mutationTable/column/AnnotationColumnFormatter.tsx

```diff
@@ -185,6 +224,12 @@ export default class AnnotationColumnFormatter
                         is3dHotspot={annotation.is3dHotspot}
                     />
                 </If>
+                <If condition={columnProps.enableCivic || false}>
+                    <Civic
+                        civicEntry={annotation.civicEntry}
+                        isCivicDisabled={annotation.isCivicDisabled}
```

Comment:
To be consistent with the rest of the implementation, I think we should not render the Civic component at all if it is disabled. As far as I understand, current implementation shows a "disabled icon" in case civic is disabled. We prefer to show nothing in that case. It makes the implementation simpler.